includes/constants.php

```diff
@@ -34,9 +34,9 @@
 	'ACTIVITYPUB_MASTODON_HTML_SANITIZER',
 	array(
 		'p'          => array(),
-		'span'       => array( 'class' ),
+		'span'       => array_flip( array( 'class' ) ),
```

Comment:
```suggestion
		'span'       => array( 'class' => true ),
```
Let's avoid the function calls

Reply:
They were already merged in #1125. I'll revert it here.